frappe/public/js/frappe/views/reports/query_report.js

```diff
@@ -1467,75 +1467,55 @@ frappe.views.QueryReport = class QueryReport extends frappe.views.BaseList {
 			return;
 		}
 
-		let export_options = ["Excel"];
-		if (this.datatable) {
-			export_options.push("CSV");
-		}
```

Comment:
Wasn't there in the original PR, need to check when it was removed on v15, incase there's other things that accompany as well.

Reply:
https://github.com/frappe/frappe/pull/25562#issuecomment-2009419466